.gitignore

```diff
@@ -1,11 +1,28 @@
-**/.nyc_output/
-**/coverage/
-**/node_modules/
-**/tmp/
-/lerna-debug.log
-package-lock.json
-
-# Editors
-.idea
-.vscode
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Dependency directory
+node_modules
+
+# Optional npm cache directory
+.npm
+
+# OS stuff
+._*
 .DS_Store
```

Comment:
by any chance can you restore `.idea` and `.vscode`, 

https://code.visualstudio.com/
https://www.jetbrains.com/idea/